### PR TITLE
added mentee registration actions and starter reducer file

### PR DESCRIPTION
**Pull request by LittleBoGeek**
Created a folder containing mentee registration actions and the skeleton of a reducer file.

### CODE REVIEW

File: src/redux/actions/MenteeRegistration.redux/reducer.js [hunk 1]
```diff
@@ -0,0 +1 @@
+initialState = {};
```
Comment (PaulMEdwards) [inline src/redux/actions/MenteeRegistration.redux/reducer.js:1]: Ignoring the fact that this does nothing and is not used anywhere, it needs to be prefixed with `var`, `let`, or preferrably `const`.

File: src/redux/actions/MenteeRegistration.redux/MenteeRegistration.actions.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+export const POST_FORM_DATA = 'POST_FORM_DATA';
```
Comment (PaulMEdwards) [inline src/redux/actions/MenteeRegistration.redux/MenteeRegistration.actions.js:1]: `POST_FORM_DATA` doesn't appear to be used anywhere.

File: src/redux/Registration.redux/Registration.actions.js [hunk 3]
```diff
@@ -12,7 +12,6 @@ import {
 import { USER_TYPES } from '../User.redux/User.types';
 
 export const SET_REGISTRATION_FORM = 'SET_REGISTRATION_FORM';
-
```
Comment (PaulMEdwards) [inline src/redux/Registration.redux/Registration.actions.js:15]: Probably shouldn't remove this whitespace... It's more clear with the whitespace that the comment block pertains to the code block below.

**Review by PaulMEdwards — CHANGES_REQUESTED** [review]
The `MenteeRegistration.redux` folder should be 1 folder up as a child of the `redux` folder rather than the `actions` folder it's presently in... Please correct that.

File: src/redux/actions/MenteeRegistration.redux/reducer.js (hunk 1 above)
Comment (PaulMEdwards) [reply src/redux/actions/MenteeRegistration.redux/reducer.js:1]: Should probably rename this file from `reducer.js` to `MenteeRegistration.reducer.js` to conform to the pattern already established in other `.redux` folders.

**Review by PaulMEdwards — CHANGES_REQUESTED** [review]

File: src/redux/Registration.redux/Registration.actions.js (hunk 3 above)
Comment (LittleBoGeek) [reply src/redux/Registration.redux/Registration.actions.js:15]: You're 100% right, that was a dumb accident.

**Review by Brodt7258 — APPROVED** [review]

**Review by PaulMEdwards — CHANGES_REQUESTED** [review]
Please reference your new `MenteeRegistration.actions.js` file in the `/redux/actions/index.js` file.

**Review by PaulMEdwards — APPROVED** [review]
Nice work; thanks!

**Pull request merged**